js/packages/web/src/views/auctionCreate/AuctionItemCard.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { ArtCard } from '../../components/ArtCard';
+import PackCard from "../../components/PackCard";
+import { useMeta } from '../../contexts';
+import { SafetyDepositDraft } from '../../actions/createAuctionManager';
+
+interface IAuctionItemCard {
+  current: SafetyDepositDraft;
+  isSelected?: boolean;
+  onSelect: () => void;
+  onClose?: () => void;
+}
+
+const AuctionItemCard = ({ current, isSelected, onSelect, onClose }: IAuctionItemCard) => {
+  const {packs, vouchers} = useMeta();
+  const masterEdition = current.masterEdition?.pubkey;
+  const voucher = Object.values(vouchers).find(v => v?.info?.master === masterEdition);
```

Comment:
Let's hide it under the env variable and please test it by removing ENABLE_NFT_PACKS from .env

```ts
const shouldShowPacks = process.env.NEXT_ENABLE_NFT_PACKS === 'true';
if (shouldShowPacks) {
  const voucher = Object.values(vouchers).find(v => v?.info?.master === masterEdition);
  if (voucher) { 
     ... 
  }
}
```